### DISCUSSION TITLE
[ERR] Could not run nmap command: exit status 255 [FTL] Could not run enumeration: Could not run nmap command: exit status 255

### DISCUSSION THREAD

**Discussion by ddawxaf** [hktalent/scan4all#152] [General]
[ERR] Could not run nmap command: exit status 255
[FTL] Could not run enumeration: Could not run nmap command: exit status 255

**Comment by hack0ne** [comment]
upgrade nmap version 7.92

**Reply by XHX836161547** [reply]
thank you , it is useful

**Comment by XHX836161547** [comment]
> [ERR] Could not run nmap command: exit status 255 [FTL] Could not run enumeration: Could not run nmap command: exit status 255

do you solve it?  i try to upgrade nmap to 7.92 but warn" Version '7.92' for 'nmap' was not found" , my nmap version is 7.8 now
